generators/client/templates/angular/_package.json

```diff
@@ -121,6 +121,7 @@
     "<%= module.name %>": "<%= module.version %>",
     <%_ }); _%>
     "uglifyjs-webpack-plugin": "1.1.4",
+    "uglify-es": "3.2.1",
```

Comment:
**Yarn** doesn't care about that (see Deepu's comment: https://github.com/jhipster/generator-jhipster/issues/6910#issuecomment-353992198)

This is for NPM only. With that:
```
09:28:07 in ~/tmp/29-npm on  master 
➜ npm ls uglify-es
jhipster@0.0.0 /home/pgrimaud/tmp/29-npm
├── uglify-es@3.2.1 
└─┬ uglifyjs-webpack-plugin@1.1.4
  └── uglify-es@3.2.1  deduped
```